### PR TITLE
feat: add tag and query API for cross stake reward

### PR DESCRIPTION
**Pull request by Pythonberg1997**
### Description

Add a tag for recon service and query api for user to get their cross stake reward by their bsc address

### Rationale

N/A

### Example

N/A

### Changes

- Add `QueryCrossStakeRewardByBscAddress` method
- Adjust total reward distrituion tag

### CODE REVIEW

File: x/stake/keeper/distribute_sidechain.go [hunk 1]
```diff
@@ -196,7 +196,7 @@ func (k Keeper) DistributeInBreathBlock(ctx sdk.Context, sideChainId string) sdk
 			if k.AddrPool != nil {
 				k.AddrPool.AddAddrs(changedAddrs[:])
 			}
-			rewardSum += totalReward
+			rewardSum += remainReward.RawInt()
```
Comment (forcodedancing) [inline x/stake/keeper/distribute_sidechain.go:199]: Not get it. To fix a known issue?

File: x/stake/client/cli/query_sidechain.go [hunk 2]
```diff
@@ -738,6 +739,56 @@ func GetCmdQuerySideAllValidatorsCount(cdc *codec.Codec) *cobra.Command {
...
+			if err != nil {
+				return err
+			}
+
+			switch viper.Get(cli.OutputFlag) {
+			case "text":
+				fmt.Println("The reward balance is:", string(response))
+			case "json":
+				//TODO
+				fmt.Println("The reward balance is:", string(response))
```
Comment (forcodedancing) [inline x/stake/client/cli/query_sidechain.go:780]: Can we return a simple json string here?

File: x/stake/keeper/distribute_sidechain.go (hunk 1 above)
Comment (Pythonberg1997) [reply x/stake/keeper/distribute_sidechain.go:199]: Yes. The recon service will compare between this with the sum of reward_details. And the reward_detail doesn't contain the validator's part. So I subtract that part from the total reward too.

File: x/stake/client/cli/query_sidechain.go (hunk 2 above)
Comment (Pythonberg1997) [reply x/stake/client/cli/query_sidechain.go]: I simplify this part by remove the output format option

**Review by forcodedancing — APPROVED** [review]

**Pull request merged**